src/router/bundle/bundleHandler.ts

```diff
@@ -58,17 +58,100 @@ export default class BundleHandler implements BundleHandlerInterface {
         this.validators = validators;
     }
 
-    /* eslint-disable  @typescript-eslint/no-unused-vars,prettier/prettier */
     async processBatch(
```

Comment:
Most of the logic in this method looks similar as processTransaction, try re-using existing logic and avoid writing the same thing twice.

Reply:
Looks good!